plan.go

```diff
@@ -62,12 +61,12 @@ type Plan struct {
 
 // PlanStatusTimestamps holds the timestamps for individual plan statuses.
 type PlanStatusTimestamps struct {
-	CanceledAt      time.Time `json:"canceled-at"`
-	ErroredAt       time.Time `json:"errored-at"`
-	FinishedAt      time.Time `json:"finished-at"`
-	ForceCanceledAt time.Time `json:"force-canceled-at"`
-	QueuedAt        time.Time `json:"queued-at"`
-	StartedAt       time.Time `json:"started-at"`
+	CanceledAt      string `jsonapi:"attr,canceled-at"`
+	ErroredAt       string `jsonapi:"attr,errored-at"`
+	FinishedAt      string `jsonapi:"attr,finished-at"`
+	ForceCanceledAt string `jsonapi:"attr,force-canceled-at"`
+	QueuedAt        string `jsonapi:"attr,queued-at"`
+	StartedAt       string `jsonapi:"attr,started-at"`
```

Comment:
This is fixed [here](https://github.com/hashicorp/jsonapi/pull/1), will update this comment soon.